packages/apps-config/package.json

```
@@ -26,6 +26,7 @@
     "@sora-substrate/type-definitions": "1.4.1",
     "@subsocial/types": "0.5.9-dev.2",
     "@zeitgeistpm/type-defs": "0.2.5",
+    "@zeroio/type-definitions": "^0.0.5",
```

```suggestion
    "@zeroio/type-definitions": "0.0.5",
```

We have had some issues with `^` dependencies - a number of projects are actually using `@polkadot/apps-config` as stadnd-alone.